packages/vite/src/plugins/plugin.ts

```diff
@@ -92,16 +92,20 @@ async function buildViteTargets(
   options: VitePluginOptions,
   context: CreateNodesContext
 ) {
+  const absoluteConfigFilePath = joinPathFragments(
+    context.workspaceRoot,
+    configFilePath
+  );
   const { loadConfigFromFile } = await loadViteDynamicImport();
   const viteConfig = await loadConfigFromFile(
```

Comment:
Is this still using `loadConfigFromFile` rather than `resolveConfig`?

Reply:
I went back to it since Qwik uses hooks for vite which would be unavailable if i used `loadFromConfigFile`